ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/KsqlAuthorizationProviderHandler.java

```diff
@@ -23,9 +23,12 @@
 import io.vertx.core.WorkerExecutor;
 import io.vertx.ext.auth.User;
 import io.vertx.ext.web.RoutingContext;
-import java.security.Principal;
 import java.util.Set;
 
+/**
+ * Handler that calls a KsqlAuthorizationProvider plugin that can be used for custom authorization
+ * (e.g. RBAC in Confluent platform)
```

Comment:
We generally avoid reference to RBAC / CP in the OSS ksqlDB code base.

Reply:
ack